Fireworks/Geometry/python/dumpRecoGeometry_cfg.py

```diff
@@ -35,18 +35,13 @@ def recoGeoLoad(score):
 
     elif score == "2017":
        from Configuration.AlCa.GlobalTag import GlobalTag
-       process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgrade2017', '')
+       process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:mc', '')
```

Comment:
@jshlee - this will pickup run1 geometry and not 2017. Please, discard this change.